src/components/Global/MoreInfo/index.tsx

```
@@ -81,7 +81,7 @@ const MoreInfo = ({ text, html = false }: MoreInfoProps) => {
                     className="whitespace-pre-line rounded border border-n-1 bg-white px-3 py-2 shadow-lg"
                 >
                     <div className="block text-h8 font-normal text-black">
-                        {html ? <div dangerouslySetInnerHTML={{ __html: text }} /> : <div>{text}</div>}
+                        {html ? <div dangerouslySetInnerHTML={{ __html: text as string }} /> : <div>{text}</div>}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)